api/python/cell_census/src/cell_census/_open.py

```diff
@@ -42,54 +51,52 @@ def open_soma(
     uri: Optional[str] = None,
     context: Optional[soma.options.SOMATileDBContext] = None,
 ) -> soma.Collection:
-    """
-    Open the Cell Census by version or URI, returning a ``soma.Collection`` containing the
-    top-level census.  Raises error if ``census_version`` is specified and unknown, or
-    if neither ``uri`` or ``census_version`` are specified, or if the ``uri`` can not
-    be opened [lifecycle: experimental].
-
-    Parameters
-    ----------
-    census_version : ``Optional[str]``
-        The version of the Census, e.g., "latest"
-    uri : Optional[str]
-        The URI containing the Census SOMA objects. If specified, will take precedence
-        over ``census_version`` parameter. An exception will be raised if the URI can
-        not be opened.
-
-    Returns
-    -------
-    ``soma.Collection`` : returns a SOMA Collection object. Can be used as a context manager, which
-        will automatically close upon exit.
-
-    Examples
-    --------
-    Open the default Cell Census version, using a context manager which will automatically
-    close the census upon exit of the context.
-
-    >>> with cell_census.open_soma() as census:
-            ...
+    """Open the Cell Census by version or URI.
 
-    Open and close:
+    [lifecycle: experimental]
 
-    >>> census = cell_census.open_soma()
-        ...
-        census.close()
+    Args:
+        census_version: The version of the Census, e.g. "latest".
+        uri: The URI containing the Census SOMA objects. If specified, will take precedence
+            over ``census_version`` parameter.
+        context: a custom :class:`SOMATileDBContext`.
 
-    Open a specific Cell Census by version:
+    Returns:
+        A SOMA Collection object containing the top-level census.
+        It can be used as a context manager, which will automatically close upon exit.
 
-    >>> with cell_census.open_soma("2022-12-31") as census:
-            ...
+    Raises:
+        ValueError: if the census cannot be found, the URI cannot be opened, or neither a URI
```

Comment:
Should the `ValueError` have a `:class:` tag?

Reply:
Not necessary - Sphinx can parse this and add the hyperlink without additional annotations.